package.json

```diff
@@ -35,7 +35,7 @@
     "ios": "react-native run-ios",
     "lint": "eslint .",
     "localize": "node ./localizeLanguage.js; git add ./src/locales/strings",
-    "precommit": "npm run localize && lint-staged && flow && npm test",
+    "precommit": "yarn-deduplicate && npm run localize && lint-staged && flow && npm test",
```

Comment:
Please move this to another hook. The precommit hook should only verify things, not edit things. Maybe prepare or fix?